content/md/products-super-power/how-to-manage-my-publications.md

```diff
@@ -45,77 +52,93 @@ To check which products are published:
+When you own the ongoing published product, there are two actions available in the header on the right side:
 
-![Edit working copy or Unpublish buttons](../img/Products-PublishedProduct2.png)
+*   Edit the current product version (working copy): clicking on this button will redirect you to the unpublished product form.
 
-*   Unpublish
+![Edit working copy or Unpublish buttons](../img/Products-PublishedProduct2.png)
 
+*   Unpublish: to unpublish this product
 
 # Unpublish products
 
-Unpublishing products remove the product’s version from being used in the exports.
+Unpublishing products remove the product’s version from being used in the published product exports.
 
-Unpublishing products only impacts the export of products. The export of attributes, categories, etc... does not take into account the concept of unpublishing product.
+Unpublishing products only impact the export of published products. The export of attributes, categories, etc... does not take into account the concept of unpublishing product.
 
 ## Unpublish a single product
 
-To unpublish one product:
-1.  Go to the published product grid by clicking on the `...` in the top right corner of the product grid screen
-1.  Click on the small cloud icon displayed on product rollover in the grid (see screenshot below)
+You can unpublish a product from the main product form or from the published product grid.
+
+From the published product grid:
+1.  Go to the published product grid (click on the `...` on the top right corner of the product grid screen)
+1.  Click on the small papersheet icon displayed on product rollover in the grid (see screenshot below)
+
+:::info
+You can also unpublish the product directly from the published product form.
+::
 
 ![Unpublished a product from the grid](../img/Products-PublishedProduct3.png)
 
-The product will be no longer published. The product will not be listed anymore in the published product grid.
+From the product grid:
+1.  Go to the product grid
+1.  Select the product to unpublish
+1.  Click on the `...` on the top right corner, 
+1.  Click on `Unpublish`
+
+The product will no longer be published and the product will not be listed anymore in the published product grid.
 
 ## Unpublish several products at once
 
 To unpublish several products at the same time:
-1.  Go to the Product grid
-1.  Click on the "..." in the top right corner of the screen
-1.  Select `Published products` in the dropdown
-1.  Tick the products in the grid
-1.  Click on the Bulk Actions button at the bottom of the screen, and select the unpublish operation
+1.  Go to the published product grid (click on the `...` in the top right corner of the product grid screen)
+1.  Select published products in the grid, the bulk action menu will appear at the bottom of the screen
+1.  Click on `Bulk action`
+1.  Then click on `Unpublish` button
 
 ![Bulk Actions on Publish products](../img/Products-PublishedProduct4.png)
 
 1.  Select the `Unpublish` operation:
```

Comment:
This line appears as item number 1 as it must be a 5 item.

Reply:
I've tried to fix it, not sure it worked :/

Reply:
Yes that's great! It worked!